src/app/views/overview/overview.component.html

```diff
@@ -65,14 +65,14 @@ <h2>Overview</h2>
                 <li>
                     <a href="https://github.com/deniszholob/factorio-cheat-sheet/issues" target="_blank" rel="noopener">
                         Known Issues
-                    </a>
+                    </a>.
                 </li>
             </ul>
 
             <p>
-                You can also <a href="assets/Factorio_Cheat_Sheet_v0.16.pdf" target="_blank" rel="noopener">download the
```

Comment:
There is no 0.17 pdf yet, so this will not work, id keep this as is, and create a new ticket for updating the pdf, and also another ticket modifying the bottom section with the previous cheatsheets to add 0.16 pdfs and link to v016.factoriosheetsheet.com there as well

Reply:
I opened new issue for a new 0.17 .PDF file generation.

I'm not sure what you mean with another ticket for the bottom section, so can you open a issue for that yourself?